src/public/curriculum/stretching-and-shrinking/stretching-and-shrinking.json

```diff
@@ -27,7 +27,16 @@
         "width": 25,
         "height": 25
       }
-    ]
+    ],
+    "showCommentTag": true,
+    "commentTags": {
+      "tag0": "Part-to-Part",
+      "tag1": "Part-to-Whole",
```

Comment:
This was added to test the deployment branch. Should we take it out since usually
`https://collaborative-learning.concord.org/branch/master/....` would point to the "clue-curriculum" repository

Reply:
I will leave it in the PR for now, since there's no way we can test it, unless we change something in the clue-curriculum repository.